vita3k/gui/src/ime.cpp

```diff
@@ -450,7 +451,7 @@ void draw_ime(Ime &ime, EmuEnvState &emuenv) {
         if (ImGui::Button("S/K", PUNCT_BUTTON_SIZE))
             ImGui::OpenPopup("S/K");
         if (ImGui::BeginPopup("S/K", ImGuiWindowFlags_NoMove)) {
-            ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%s", "Change language of keyboard");
+            ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%s", gui.lang.ime["change_language_of_keyboard"].c_str());
             ImGui::Spacing();
             ImGui::Separator();
             ImGui::Spacing();
```

Comment:
sorry, my fault, remove this, it njo exist att all, it just display langue of keyboard

Reply:
done